demos-europe/demosplan-core
# B dplan 13034 use api2 to create user

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-13034

- Move to using AdministratableUserResourceType instead of old 1.0 api
- Important things to check: 
- Updating Roles,Department and Orga relationships
- When a user is created, login field is set same as email field

### How to review/test
1. Create a user and check that emails has been sent (not possible to test in local)
2. Update user all possible combinations: roles, orga, email.
3. when updating email, check that login is also updated


Delete the checkbox if it doesn't apply/isn't necessary. -->
